src/fstalign.cpp

```diff
@@ -206,6 +209,21 @@ vector<shared_ptr<Stitching>> make_stitches(spWERA alignment, vector<RawCtmRecor
       }
     }
 
+    if (!one_best_tokens.empty()) {
+      auto token = one_best_tokens[hypRowIndex];
+      part->hyp_orig = token;
+
+      // sanity check
+      std::string token_copy = token;
+      std::transform(token_copy.begin(), token_copy.end(), token_copy.begin(), ::tolower);
+      if (hyp_tk != token_copy) {
+        logger->warn(
+            "hum, looks like the text and the alignment got out of sync? [{}] vs "
+            "[{}]",
+            hyp_tk, token_copy);
+      }
```

Comment:
Is this needed or was it for debugging purposes?

Reply:
just to match the ctm logging above